pkg/backup/backup_compaction_test.go

```diff
@@ -315,6 +315,63 @@ crdb_internal.json_to_pb(
 	})
 	// TODO (kev-cao): Once range keys are supported by the compaction
 	// iterator, add tests for dropped tables/indexes.
+	t.Run("automatic compaction after scheduled backup", func(t *testing.T) {
```

Comment:
I'm thinking a test for encrypted scheduled backups is in order here, since in my own testing, this was a problem area that I had to deal with several times (although that's mostly to do with our `nil` pointer vs `EncryptionMode_None` issue).